HJM.Chip8/HJM.Chip8.CPU/Chip8.cs

```diff
@@ -101,400 +107,33 @@ public void EmulateCycle()
         {
             // Fetch Opcode
             // Convert 2 1 byte memory addresses to 1 2 byte op code
```

Comment:
Now as you're in cleaning mode, maybe also move this to some helper method?

Consider also moving steps in this method to own methods. It's somewhat of antipattern I've learned to notice that if you need to label code with comments (in this case "Execute OpCode", "Update timers", "play sound"...) those should be moved to their methods with that named with that comment (ExecuteOpCode, UpdateTimers, PlaySound).

Reply:
I agree, those are just leftovers from the guide I followed originally. Now that the project is getting bigger I think that does make sense.